cmd/root.go

```diff
@@ -10,6 +10,10 @@ import (
 
 var cfgFile string
 
+// SilentMode represents whether or not the program is running in silent mode.
+// Silent mode causes the program to throw an error when config values are missing.
+var SilentMode bool
```

Comment:
You don't need this with Viper.

Reply:
Thought I removed this since it isn't used anymore. Good catch